src/fpm_global.f90

```diff
@@ -0,0 +1,7 @@
+    module fpm_global
```

Comment:
Do we need to introduce this global state? I thought the existing method for specifying verbosity via the command line settings structure was sufficient and we can pass verbosity as an subroutine argument wherever it's needed. The global state breaks modularity which is useful for unit testing and using *fpm* as a package dependency.

Reply:
I new module for things like types and options like verbosity allows building with the Intel compiler in particular; I have a version where moving a few things into the new module allows building with ifort for one. Played with three different ways and passing the value around seemed tedious.  I will try a different way; did not realize it would cause a problem. In my own build system everything gets logged whether it goes to the screen or not, so that seems like a good idea but not sure how it should be implemented. log everything and refer to it on error; or display it if an error occurs, etc. Part of the problem is the gfortran options used by default produce a lot of superfluous messages. Looks very different if you build with "-profile release", for instance. Also had some odd output when colored on a ming64 terminal, but not a ming32 terminal; worked as expected everywhere else I tried; the error stop thing is something I never cared for much but was occasionally useful when developing fpm itself; not particularly useful. Maybe leave it as an option that gets used if an environment variable like $DEBUG is set? Otherwise, removing ERROR STOP is fine by me. You want the location to always show or just if verbose mode is on?